db/external_sst_file_ingestion_job.cc

```diff
@@ -609,14 +609,12 @@ Status ExternalSstFileIngestionJob::GetIngestedFileInfo(
 
     // Set the global sequence number
     file_to_ingest->original_seqno = DecodeFixed64(seqno_iter->second.c_str());
-    auto offsets_iter = props->properties_offsets.find(
-        ExternalSstFilePropertyNames::kGlobalSeqno);
-    if (offsets_iter == props->properties_offsets.end() ||
-        offsets_iter->second == 0) {
+    if (props->external_sst_file_global_seqno_offset == 0) {
       file_to_ingest->global_seqno_offset = 0;
       return Status::Corruption("Was not able to find file global seqno field");
     }
-    file_to_ingest->global_seqno_offset = static_cast<size_t>(offsets_iter->second);
+    file_to_ingest->global_seqno_offset =
```

Comment:
Note to reviewer:
There has been discussion on replacing `properties_offsets` with a bool flag indicating the existence of external sst file global seqno. However, it seems that we need to add `global_seqno_offset` to `file_to_ingest` here for ingestion later, which in turns require us to keep the offsets.

Reply:
Sounds good.